File: app/src/main/java/com/example/androidbasic/ui/activity/HomeActivity.java
package com.example.androidbasic.ui.activity;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;

/**
 * Created by vikas on 26/9/16.
 */

public class HomeActivity extends BaseActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Intent intent = new Intent(this,ActivityBasicActivity.class);
        startActivity(intent);
    }
}

File: app/src/main/java/in/vikasrathod/androidbasic/ui/activity/HomeActivity.kt
package `in`.vikasrathod.androidbasic.ui.activity

import android.content.Intent
import android.os.Bundle

/**
 * Created by vikas on 26/9/16.
 */

class HomeActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        startActivity(Intent(this,ToastActivity::class.java))
    }
}
